### ISSUE TITLE
Mask RCNN, HTC models, why only backbone has norm_cfg

### ISSUE THREAD

**Issue by minan19605** [open-mmlab/mmdetection#3674]
I checked config files for mask RCNN, HTC,  I found only the backbone part has norm_cfg, could you explain why? 
Why FCN mask head don't have norm_cfg ?

Thanks a lot!

model = dict(
    type='HybridTaskCascade',
    pretrained='torchvision://resnet50',
    backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=1,
        **_norm_cfg=dict(type='BN', requires_grad=True),_**
        norm_eval=True,
        style='pytorch'),

**Comment by minan19605** [comment]
I also found config in "gn" and "gn+ws", In those config files, all parts have norm_cfg.

norm_cfg = dict(type='GN', num_groups=32, requires_grad=True)
model = dict(
    pretrained='open-mmlab://detectron/resnet50_gn',
    backbone=dict(norm_cfg=norm_cfg),
    neck=dict(norm_cfg=norm_cfg),
    roi_head=dict(
        bbox_head=dict(
            type='Shared4Conv1FCBBoxHead',
            conv_out_channels=256,
            norm_cfg=norm_cfg),
        mask_head=dict(norm_cfg=norm_cfg)))

**Comment by ZwwWayne** [comment]
This is a common practice. For mask r-cnn, faster r-cnn, and retinanet, they only contain BN in backbone and do not use BN in the head. If you try to add BN in the head, the performance will drop since the batch size is small. However, you could also try to use `GN` in the head like FCOS, which could improve performance.

**Comment by minan19605** [comment]
Thanks a lot for your comments

**Comment by zhangshen12356** [comment]
> This is a common practice. For mask r-cnn, faster r-cnn, and retinanet, they only contain BN in backbone and do not use BN in the head. If you try to add BN in the head, the performance will drop since the batch size is small. However, you could also try to use `GN` in the head like FCOS, which could improve performance.

hi, i want to add shared_head in htc_roi_head, but _**assert not self.with_shared_head  # shared head is not supported**_ in htc_roi_head.  So  if i add the shared_head what will happen?
